voicevox/speaker_info.py

```diff
@@ -0,0 +1,59 @@
+# voicevox - speaker_info
+
+from typing import List
+
+from .types.speaker_info import SpeakerInfoType, StyleInfoType
+
+class StyleInfo:
+    """Return style info
+
+    Attributes
+    ----------
+    id: int
+        style id
+    icon: str
+        base64 encoded icon
+    portrait: str
+        base64 encoded portrait image
+    voice_samples: list[str]
+        raw voice_data ???
```

Comment:
Why here are question mark?

Reply:
Sorry at the time I wrote that I didn't know the exact return type, now I tested it and corrected it
it was base64 encoded audio all along.